tests/UI/pages/BO/customerService/orderMessages/add.js

```diff
@@ -10,11 +10,11 @@ module.exports = class AddOrderMessage extends BOBasePage {
 
     // Selectors
     this.nameLangButton = '#order_message_name';
-    this.langDropdownDiv = 'div.dropdown-menu[aria-labelledby=\'order_message_name\']';
-    this.nameLangSpan = lang => `${this.langDropdownDiv} span[data-locale='${lang}']`;
+    this.langDropdownDiv = 'div.locale-dropdown-menu';
+    this.nameLangSpan = lang => `${this.langDropdownDiv} span.locale-iso-code-${lang}`;
```

Comment:
not really useful because there is already a data selector :thinking:

Reply:
We want to improve all kind of selectors that not use id or a specific class

Reply:
What data selector are you referring to ?